OpenRA.Mods.Common/Traits/Crates/Crate.cs

```diff
@@ -107,13 +107,13 @@ void INotifyParachute.OnLanded(Actor self, Actor ignore)
 			var collector = landedOn.FirstOrDefault(a =>
 			{
 				// Mobile is (currently) the only trait that supports crushing
-				var mi = a.Info.TraitInfoOrDefault<MobileInfo>();
+				var mi = a.TraitOrDefault<Mobile>();
```

Comment:
Why change from `TraitInfoOrDefault<MobileInfo>()` to `TraitOrDefault<Mobile>().Info`?  If we want to keep this, then the `mi` should change to `mobile`.